pwa/urls.py

```diff
@@ -1,10 +1,23 @@
-from django.urls import re_path
+from django import VERSION
+
 
 from .views import manifest, service_worker, offline
 
-# Serve up serviceworker.js and manifest.json at the root
-urlpatterns = [
-    re_path(r'^serviceworker\.js$', service_worker, name='serviceworker'),
-    re_path(r'^manifest\.json$', manifest, name='manifest'),
-    re_path('^offline/$', offline, name='offline')
-]
+if VERSION >= (4, 0):
```

Comment:
`path` exists since Django 2.0. I don't think this condition is necessary.

I do think it would be more helpful for the project to drop support for unsupported Django versions.
We should update `setup.py` to Django 2.0+ (or even 3.2+).